packages/edit-site/src/components/sidebar/global-styles-v2/navigation/view.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import {
+	VIEW_COLORS,
+	VIEW_COLORS_ELEMENT,
+	VIEW_COLORS_PALETTE,
+	VIEW_ROOT,
+	VIEW_TYPOGRAPHY,
+	VIEW_TYPOGRAPHY_ELEMENT,
+} from './constants';
+import { useGlobalStylesNavigationContext } from './context';
+import GlobalStylesViewColors from '../views/colors';
+import GlobalStylesViewColorsElement from '../views/colors-element';
+import GlobalStylesViewColorsPalette from '../views/colors-palette';
+import GlobalStylesViewRoot from '../views/root';
+import GlobalStylesViewTypography from '../views/typography';
+import GlobalStylesViewTypographyElement from '../views/typography-element';
+
+const allViews = {
```

Comment:
This may grow huge with all the subviews. Maybe the root view does not need to be aware of the subviews and just of its children, and then its children are aware of its subviews.
Another alternative is importing the subviews from the children.
```
import { views as colorViews } from '../views/colors'

const allViews = {
	...colorViews,
}
```
But for now keeping them in a central place is ok we don't know yet if it is going to be a problem or not.

Reply:
Absolutely. Once we're done with this, it will have to be dynamic, otherwise it's just useless.